.storybook/custom/custom.scss

```diff
@@ -194,17 +223,3 @@ body {
     width: 100%;
     padding: 10px 10px;
 }
-
-.break {
-    flex-basis: 100%;
-    height: 0;
-    border-bottom: 10px solid transparent;
-}
-
-.demo-icon-wrapper {
```

Comment:
These were needed! They help with layouts - that's why you had to update snaps. Check out icons: https://deploy-preview-1018--fundamental-react.netlify.app/?path=/docs/component-api-icon--primary

Reply:
ok, i see that `.demo-icon-wrapper` is needed. 

Is `.break` needed too?